Initial commit of text for intro (#11)
greenelab/covid19-review: content/02.introduction.md

## Introduction {.page_break_before} 

> ### General Background

Reviewer: ```suggestion

### General Background

```
Add blank lines to match other section headers.

> On January 21, 2020, the World Health Organization (WHO) released its first report concerning what is now known as the Coronavirus disease 2019 (COVID-19) [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4].

Reviewer 2: The citation keys don't support the `=` character.  You could use a citation tag or simply remove `?sfvrsn=20a99c10_4` from this citation and others like it.

Author: Thank you so much! I'll fix that!

> This infections disease came to international attention on December 31, 2019 following an announcement by national officials in China about 44 cases of a respiratory infection of unknown cause.

Reviewer: ```suggestion
This infectious disease came to international attention on December 31, 2019 following an announcement by national officials in China about 44 cases of a respiratory infection of unknown cause.
```

> The first known cases were centralized in Wuhan City within the Hubei province of China, but the diease subsequently began to spread rapidly beyond Wuhan within China and around the world. 

Reviewer: ```suggestion
The first known cases were located in Wuhan City within the Hubei province of China, but the disease subsequently began to spread rapidly beyond Wuhan within China and around the world. 
```

> At the time of the first situation report [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4], 282 confirmed cases had been identified, primarily in China, but also the first 1 to 2 cases in each of Thailand, Japan, and the Republic of Korea.

Reviewer: ```suggestion
At the time of the first situation report [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4], 282 confirmed cases had been identified, primarily in China, but also the first 1 to 2 cases had been found in each of Thailand, Japan, and the Republic of Korea.
```

> One week later, 4593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe [@urL:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf?sfvrsn=8b671ce5_2].

Reviewer: ```suggestion
One week later, 4593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf?sfvrsn=8b671ce5_2].
```

Other well-known coronaviruses include those associated with previous infectious diseases of global concern, including Severe Accute Repiratory Syndrome (SARS) and Middle East respiratory syndrome (MERS); however, some coronaviruses are associated with less virulent illnesses including the common cold.
The SARS-CoV-2 virus was unknown until approximately January 12, 2020, when Chinese officials released its genetic sequence to aid in worldwide efforts to diagnose the disease [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf?sfvrsn=20a99c10_4].
As researchers worldwide work to characterize SARS-CoV-2 and COVID-19, information about the transmission and life cycle of the virus as well as the diagnosis and treatment of the disease is emerging rapidly.
In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.

### Coronaviruses: What are they, and what do we know about SARS-CoV-19?

> Coronaviruses are RNA viruses that...

Reviewer: Do you want to include mechanisms of entry into cells here? Or would it go below? I don't think it needs to be written but I'm not sure where it would go. A lot of folks are thinking about ACE2, so having some discussion somewhere is probably good.

Author: Should be addressed in commit 40b872b below

### Mechanisms of Coronavirus-driven Disease in Humans

Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...

### Presentation of COVID-19

Information is rapidly becoming available about the wide range of symptoms that can be associated with COVID-19 as well as the range of symptom severity, onset from exposure, and possible risk or protective factors...

### Vaccines for Viruses: Strategies for and challenges to development

What information is needed to develop a vaccine? How have vaccines for other viruses such as H1N1 been developed?

### Diagnostics and Therapeutics for Viruses

Two major concerns within diagnosis include the detection of current infections in individuals with and without symptoms, and the detection of past exposure without a live infection.

> In the latter category, identifying whether individuals can develop sustained immunity is also a major consideration.

Reviewer: ```suggestion
In the latter category, identifying whether individuals can develop or have developed sustained immunity is also a major consideration.
```
